### ISSUE TITLE
Tron 2.0 (327740)

### ISSUE THREAD

**Issue by Bumbadawg** [ValveSoftware/Proton#3117]
# Compatibility Report
- Name of the game with compatibility issues: Tron 2.0
- Steam AppID of the game: 327740

## System Information
- GPU: GTX1070
- Driver/LLVM version: 430.26
- Kernel version: 5.0.0-20-mfutex
- Link to full system information report as [Gist](https://gist.github.com/Bumbadawg/d02ccadbda9dca8fd492e79fe2b57df7):
- Proton version: 4.11-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available

[steam-327740.log](https://github.com/ValveSoftware/Proton/files/3682808/steam-327740.log)

## Symptoms 
Launcher works, starting the actual game results into a crash.

## Reproduction
Start the game, with any settings.
i used `PROTON_LOG=1 PROTON_NO_D3D9=1 PROTON_NO_FSYNC=1 %command%` and it didn't impact the failure.

**Comment by kisak-valve** [comment]
This looks like the line of interest from the log: `wine: Call from 0x7b4520df to unimplemented function msvcirt.dll.??0ifstream@@QAE@PBDHH@Z, aborting`

Upstream: https://bugs.winehq.org/show_bug.cgi?id=22616

**Comment by Bumbadawg** [comment]
yes @kisak-valve, i copied the v5 32 bit version of msvcirt.dll into the game folder and now the game launches !
i had to disable D9VK though (PROTON_USE_D9VK) because otherwise, textures are clipping into each others.

**Comment by kisak-valve** [comment]
Can't play TRON 2.0 anymore

Issue transferred from https://github.com/ValveSoftware/Proton/issues/8555.
@jebez1 posted on 2025-03-23T09:47:02:

# Compatibility Report
- Name of the game with compatibility issues: TRON 2.0
- Steam AppID of the game: 327740

## System Information
- GPU: GTX 1660 SUPER discrete
- Video driver version: 570.133.07
- Kernel version: 6.13.7-arch1-1 (64-bit)
- Link to full system information report as [Gist](https://gist.github.com/jebez1/3301ac951ec112b59f7c02bee90af5fc):
- Proton version: Hotfix

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-327740.log](https://github.com/user-attachments/files/19408112/steam-327740.log)

## Symptoms
On Arch Linux KDE.

Original or Killer App Mod.

Playing the game, there's this window (Killer App Mod installed)
![Image](https://github.com/user-attachments/assets/adf95b13-5363-499d-a2f5-10cae67ac4d8)
instead of a window with "PLAY".

It works on Windows 11.

## Reproduction
Install the game with Proton Hotfix
Play it.
